README.md

```diff
@@ -31,12 +31,16 @@ First, you need to create the OAuth 2.0 client configuration that will be used t
 
 1. [Create a new application](https://developer.factset.com/learn/authentication-oauth2#creating-an-application) on FactSet's Developer Portal.
 2. When prompted, download the configuration file and move it to your development environment.
+3. Performs authentication with FactSet's OAuth 2.0 authorization server and retrieves an access token.
```

Comment:
The content in this readme vs the text in e.g. the Java version is different.

The Java version has two sections:
* Usage - same as in this repo, the added text doesnt quote fit in here
* Modules - missing here, and that has the added text about reuse

Lets copy over that module section and change links and code examples to fit this repo.